Assets/3rd party/Standard Assets/Characters/FirstPersonCharacter/Scripts/FirstPersonController.cs

```
@@ -236,6 +258,7 @@ private void GetInput(out float speed)
 
         private void RotateView()
         {
+            if(!m_SceneController.getPauseState()) //this if statement stops the mouselook when the gameobject 'SceneControllerObject', which MUST be pressent in the scene bacause of composition, has its scenePauser make getPauseState return true or false. 

```

Why don't you check the pause state at the beginning of the update statement? This should be cleaner, as the whole script should be disabled in a pause state. It's kind of [failing fast](https://en.wikipedia.org/wiki/Fail-fast).

```
public void Update() {
  if(m_SceneController.getPauseState()) {
    return;
  }
  ...
}
```

And use curly brackets `{}` even if it's just one statement in the block, this avoid possible problems in the future and it's less confusing, especially in this case of wrong indentation.